src/kudu/consensus/peer_manager.h

```diff
@@ -71,14 +70,14 @@ class PeerManager {
  private:
   std::string GetLogPrefix() const;
 
-  typedef std::unordered_map<std::string, std::shared_ptr<Peer>> PeersMap;
```

Comment:
seems unnecessary change.

Reply:
This was previously used at some point in the rest of the code however having this typedef for a single definition (in the same header file) seemed superfluous. However you're right, this change is basically a non-sequitur.